install_script_op_worker.sh

```diff
@@ -0,0 +1,572 @@
+  worker_major_version=${DD_OP_WORKER_MAJOR_VERSION}
+fi
+
+if [ -n "$DD_OP_WORKER_MINOR_VERSION" ]; then
+  # Examples:
+  #  - 20   = defaults to highest patch version x.20.2
+  #  - 20.0 = sets explicit patch version x.20.0
+  # Note: Specifying an invalid minor version will terminate the script.
+  worker_minor_version=${DD_OP_WORKER_MINOR_VERSION}
+fi
+
+worker_dist_channel=stable
+if [ -n "$DD_OP_WORKER_DIST_CHANNEL" ]; then
+  if [ "$repository_url" == "datadoghq.com" ]; then
+    if [ "$DD_OP_WORKER_DIST_CHANNEL" != "stable" ] && [ "$DD_OP_WORKER_DIST_CHANNEL" != "beta" ]; then
+      ERROR_MESSAGE="DD_OP_WORKER_DIST_CHANNEL must be either 'stable' or 'beta'. Current value: $DD_OP_WORKER_DIST_CHANNEL"
+      echo "$ERROR_MESSAGE"
+      exit 1;
+    fi
+  elif [ "$DD_OP_WORKER_DIST_CHANNEL" != "stable" ] && [ "$DD_OP_WORKER_DIST_CHANNEL" != "beta" ] && [ "$DD_OP_WORKER_DIST_CHANNEL" != "nightly" ]; then
+    ERROR_MESSAGE="DD_OP_WORKER_DIST_CHANNEL must be either 'stable', 'beta' or 'nightly' on custom repos. Current value: $DD_OP_WORKER_DIST_CHANNEL"
+    echo "$ERROR_MESSAGE"
+    exit 1;
+  fi
+  worker_dist_channel=$DD_OP_WORKER_DIST_CHANNEL
+fi
+
+if [ -n "$TESTING_YUM_VERSION_PATH" ]; then
+  yum_version_path=$TESTING_YUM_VERSION_PATH
+else
+  yum_version_path="${worker_dist_channel}/observability-pipelines-worker-${worker_major_version}"
+fi
+
+if [ -n "$TESTING_APT_REPO_VERSION" ]; then
+  apt_repo_version=$TESTING_APT_REPO_VERSION
+else
+  apt_repo_version="${worker_dist_channel} observability-pipelines-worker-${worker_major_version}"
+fi
+
+if [ ! "$apikey" ]; then
+  if [ ! -e "$env_file" ]; then
+    printf "\033[31mAPI key not available in DD_API_KEY environment variable.\033[0m\n"
+    exit 1;
+  fi
+fi
+
+if [ ! "$op_pipeline_id" ]; then
+  if [ ! -e "$env_file" ]; then
+    printf "\033[31mPipeline ID not available in DD_OP_PIPELINE_ID environment variable.\033[0m\n"
+    exit 1;
+  fi
+fi
+
+# OS/Distro Detection
+# Try lsb_release, fallback with /etc/issue then uname command
+KNOWN_DISTRIBUTION="(Debian|Ubuntu|RedHat|CentOS|Amazon)"
+DISTRIBUTION=$(lsb_release -d 2>/dev/null | grep -Eo $KNOWN_DISTRIBUTION  || grep -Eo $KNOWN_DISTRIBUTION /etc/issue 2>/dev/null || grep -m1 -Eo $KNOWN_DISTRIBUTION /etc/os-release 2>/dev/null || uname -s)
+
+if [ "$DISTRIBUTION" = "Darwin" ]; then
+    ERROR_MESSAGE="This script does not support installing on Mac."
+    printf "\033[31m$ERROR_MESSAGE
+\033[0m\n"
+    exit 1;
+
+elif [ -f /etc/debian_version ] || [ "$DISTRIBUTION" == "Debian" ] || [ "$DISTRIBUTION" == "Ubuntu" ]; then
+    OS="Debian"
+elif [ -f /etc/redhat-release ] || [ "$DISTRIBUTION" == "RedHat" ] || [ "$DISTRIBUTION" == "CentOS" ] || [ "$DISTRIBUTION" == "Amazon" ]; then
+    OS="RedHat"
+# Some newer distros like Amazon may not have a redhat-release file
+elif [ -f /etc/system-release ] || [ "$DISTRIBUTION" == "Amazon" ]; then
+    OS="RedHat"
+fi
+
+# Root user detection
+if [ "$(echo "$UID")" = "0" ]; then
```

Comment:
```suggestion
if [ "$(echo "$UID")" == "0" ]; then
```
according to google shell [guide](https://google.github.io/styleguide/shellguide.html#testing-strings)

Reply:
Addressed in https://github.com/DataDog/agent-linux-install-script/pull/66/commits/5b76b4d796bcc42eb3364ed7cd8fc9e9a33fe997 / https://github.com/DataDog/agent-linux-install-script/pull/66/commits/707be0c4f2c8d66fe3ad78290f755dd6102a5725